improver/cli/__init__.py

```diff
@@ -225,11 +229,34 @@ def constrained_inputcubelist_converter(to_convert):
         Returns:
             iris.cube.CubeList:
                 The loaded cubelist of constrained cubes.
+
+        Raises:
+            ValueError:
+                If multiple possible constraints are provided using a list,
+                then only one constraint must be valid.
```

Comment:
Not technically your problem but I don't think the args string for this function is right.  Can you clarify it at all?

Reply:
I've amended the `args` docstring, as suggested. What do you think?

Reply:
This is clearer - thanks.